src/Utopia/Messaging/Adapter.php

```diff
@@ -248,4 +252,28 @@ protected function requestMulti(
 
         return $responses;
     }
+
+
+    /**
+     * @param string $phone
+     * @return int|null
+     * @throws Exception
+     */
+    public function getCountryCode(string $phone): ?int
+    {
+        if (empty($phone)) {
+            throw new Exception('No phone number was passed.');
+        }
+
+        $helper = PhoneNumberUtil::getInstance();
+
+        try {
+            return  $helper
+                ->parse($phone)
+                ->getCountryCode();
+
+        } catch (NumberParseException $e) {
+            throw new Exception("Error parsing phone: " . $e->getMessage());
+        }
```

Comment:
Is `NumberParseException` the only exception that can be thrown ? I think we should instead catch the Throwable?